packages/metals-vscode/src/extension.ts

```diff
@@ -722,13 +722,7 @@ function launchMetals(
             item.tooltip = params.tooltip;
           }
           if (params.command) {
-            const command = params.command;
             item.command = params.command;
-            commands.getCommands().then((values) => {
-              if (values.includes(command)) {
-                commands.executeCommand(command);
-              }
-            });
```

Comment:
I think we use this callback, e.g. `DebugProvider.reportErrors` for `DebugErrorsPresent` in metals. (I haven't tested this)

Reply:
So, this was supposed to only invoke the command if anyone clicks on the status, so I am not sure why this was done and why it only started to pop up.

Reply:
But, I think we set it to status if we don't care to show it right away, otherwise we should invoke a command instead

Reply:
![scala-debug-adapter-1691161142589](https://github.com/scalameta/metals-vscode/assets/3807253/f20b66a1-cce3-420c-9701-4c7f175e5dd1)

This started to happen :D